sdk/identity/identity/README.md

```diff
@@ -139,15 +139,15 @@ If you encounter bugs or have suggestions, please [open an issue](https://github
 
 If you'd like to contribute to this library, please read the [contributing guide](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md) to learn more about how to build and test the code.
 
-[1]: https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-identity/1.0.0/classes/defaultazurecredential.html
-[2]: https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-identity/1.0.0/classes/managedidentitycredential.html
-[3]: https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-identity/1.0.0/classes/environmentcredential.html
-[4]: https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-identity/1.0.0/classes/clientsecretcredential.html
-[5]: https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-identity/1.0.0/classes/clientcertificatecredential.html
-[6]: https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-identity/1.0.0/classes/devicecodecredential.html
-[7]: https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-identity/1.0.0/classes/authorizationcodecredential.html
-[8]: https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-identity/1.0.0/classes/interactivebrowsercredential.html
-[9]: https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-identity/1.0.0/classes/usernamepasswordcredential.html
+[1]: https://aka.ms/azsdk/js/docs/ref/azure-identity/1.0.0/classes/defaultazurecredential.html
```

Comment:
This won't work unless you create each one of these individually.  Aka doesn't retain path. It only retains # or ?

Reply:
Yes, for sure! I am assuming I'll be manually creating the ones without the hashes and the question marks and the others will work.

Reply:
https://aka.ms/azsdk/js/docs/ref/identity/defaultazurecredential is a very long short url.  I'm starting to think we need a different solution for this.  What are the other langs doing?

Reply:
I think most languages use aka only for mail links (landing page for ref docs etc). There are some instances for more detailed entities like https://aka.ms/azsdk-python-storage-blob-blobserviceclient which is effectively the same as here without docs and ref and swapping with `-`

Reply:
This PR has changed significantly thanks to our conversations! Please read the description and review it again 🙏